### ISSUE TITLE
Release v1.59.4

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#8146]
:robot: A dedicated issue to run tests and deploy release [v1.59.4](https://github.com/deckhouse/deckhouse/milestone/302).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe | be | se`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.59.4 | test-v1.59.4*` or a branch `main | release-1.59`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.25 | 1.26 | 1.27 | 1.28 | 1.29`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.59.4 | test-v1.59.4*` or a branch `main | release-1.59`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/k8s/1.27
/e2e/use/k8s/1.29

This comment will run 2 e2e jobs on AWS with containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/k8s/1.27

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.59
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [052b95](https://github.com/deckhouse/deckhouse/commit/052b95e1d3db07341c7691e1d091806545f52eab) in branch **main**:
```
[cert-manager] Actual issuers (#8149)

Signed-off-by: Dmitrii Kovalkov <dmitrii.kovalkov@flant.com>
Co-authored-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [50e949](https://github.com/deckhouse/deckhouse/commit/50e9499f9868b24016df92cc008ccaea0f51f9eb) in branch **main**:
```
[documentation] Update module version badge to display only module version instead of the dropdown list. (#8145)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8b37f0](https://github.com/deckhouse/deckhouse/commit/8b37f0c382e52cf377d6a99a6cc71ad8a74eecb3) in branch **main**:
```
[global-hooks] Fix K8s upgrade migration (#8148)

Signed-off-by: Vitaliy Snurnitsin <vitaliy.snurnitsin@flant.com>
Co-authored-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0837af](https://github.com/deckhouse/deckhouse/commit/0837af264b7989dfc3b39f7b3ce6440ec5fc295c) in branch **main**:
```
Re-generate changelog v1.59.3 (#8112)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [f9220f](https://github.com/deckhouse/deckhouse/commit/f9220fca03b4d42d8a5c2815e52bde31671e3940) in branch **main**:
```
[operator-trivy] add reports filtration (#8096)

Signed-off-by: Stepan Paksashvili <stepan.paksashvili@flant.com>
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
Co-authored-by: Stepan Paksashvili <stepan.paksashvili@flant.com>
Co-authored-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [278e2d](https://github.com/deckhouse/deckhouse/commit/278e2db0bdc7b390419002ff20f79b6cf6ad17a1) in branch **main**:
```
[docs][getting_started] Remove unnecessary nodeTemplate set (#8097)

Signed-off-by: Dmitrii Kovalkov <dmitrii.kovalkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2c2a89](https://github.com/deckhouse/deckhouse/commit/2c2a89cfeb765cffb221c64d78926dc97f893407) in branch **main**:
```
[prometheus] Switch Alertmanager from alertmanagerConfiguration to alertmanagerConfigSelector (#8158)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2bcd7c](https://github.com/deckhouse/deckhouse/commit/2bcd7cc0912e9c07ac89bdd34a14092c8473cecb) in branch **main**:
```
[dhctl] Create DeckhouseRelease for new cluster (#8143)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4dda60](https://github.com/deckhouse/deckhouse/commit/4dda6076eab9a4bca75a3ed538c077949ec38725) in branch **main**:
```
[l2-load-balancer] Fixes for ports conflict and hook panic (#8160)

Signed-off-by: Alexander Zverev <beastlex@gmail.com>
Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
Co-authored-by: Andrey.Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.59.4** is created.
